src/main/java/com/mitchej123/hodgepodge/HodgepodgeEventHandler.java

```diff
@@ -19,4 +37,105 @@ public void onZombieAidSummon(ZombieEvent.SummonAidEvent event) {
     public void setAidTriggerDisabled(boolean disableAidTrigger) {
         xuDisableAidTrigger = disableAidTrigger;
     }
+
+    @SideOnly(Side.CLIENT)
+    @SubscribeEvent(priority = EventPriority.LOWEST)
+    public void getTooltip(ItemTooltipEvent event) {
+        String producesPollutionFormat = "Produces %d Pollution/Second";
+
+        if (Common.config.furnacesPollute) {
+            String furnacePollution = String.format(producesPollutionFormat, Common.config.furnacePollutionAmount);
+            if (GT_Utility.areStacksEqual(event.itemStack, new ItemStack(Blocks.furnace))
```

Comment:
Hodgeporge doesn't require GT dependency. And IIRC this mod is also published separately from GT:NH modpack (so mod can be used in any modpack).
So to prevent crashes with `NoClassDefFoundError` when GT isn't loaded, this whole handler should be registered only when GT is loaded.

Reply:
Move the code in the client folder and only register the event bus if GT5 is present.